index.d.ts

```diff
@@ -0,0 +1,61 @@
+import { FastifyPlugin } from "fastify";
+import {
+  Connection,
+  ConnectionOptions,
+  escape,
+  format,
+  Pool,
+  PoolOptions,
+} from "mysql2";
+import {
+  Connection as PromiseConnection,
+  Pool as PromisePool,
+} from "mysql2/promise";
```

Comment:
Since this depends on external types, please update installations instructions to include them.
See "TypeScript usage" in "https://github.com/fastify/fastify-postgres/blob/master/README.md"  as a reference.

Reply:
The typing provided by the package itself, so no more guide needed in this case